tests/reg_tests/refs/error_euler_wing_partial.json

```diff
@@ -1,12 +1,12 @@
 {
     "errors": {
```

Comment:
Yes this is expected, since this PR changes the path of convergence and the test is checking some semi-converged state. This test is probably poorly written, it would better to do the following:
- partial convergence
  - compute error, then fully converge the solution and check that the error estimation is close to the real error
- full convergence
  - check errors are almost zero (we already have this)

That way the test is independent of the semi-converged state as long as we are in the linear region of convergence.